sogs/static/view_room.js

```diff
@@ -51,10 +64,51 @@ const setup = async () => {
                     {
                         throw Error(err);
                     }
-
+                    
                     const plain = Message.decode(data).dataMessage;
+                    // console.log("VERIFY:")
```

Comment:
omit such hunks, git has this very cool mode. `git add -p` that lets to step through and selectively add hunks interactively to the current commit which is amazeballs.